README.md

```diff
@@ -31,7 +31,7 @@ As of now, we have several different PHP versions. Use appropriate php version a
 * Run the `docker-compose up -d`.
 
 ```shell
-git clone https://github.com/sprintcube/docker-compose-lamp.git
+git clone https://github.com/harshalone/docker-compose-lamp.git
```

Comment:
Could you please fix this line? ;)

Reply:
Looks like this pull request is stale. Going to close this one since there is no response.